content/en/docs/ops/integrations/skywalking/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Apache SkyWalking
+description: How to integrate with Apache SkyWalking.
+weight: 32
+keywords: [integration,skywalking,tracing]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+[Apache SkyWalking](http://skywalking.apache.org) is an application performance monitor (APM) system, especially designed for
+microservices, cloud native and container-based architectures.
+
+## Installation
+
+### Option 1: Quick start
+
+Istio provides a basic sample installation to quickly get SkyWalking up and running:
+
+{{< text bash >}}
+$ kubectl apply -f @samples/addons/extras/skywalking.yaml@
+{{< /text >}}
+
+This will deploy SkyWalking into your cluster. This is intended for demonstration only, and is not tuned for performance or security.
+
+Istio proxies by default don't send traces to SkyWalking, you will also need to enable the SkyWalking tracing extension provider with
+the following fields to your configuration:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    extensionProviders:
+      - skywalking:
+          service: tracing.istio-system.svc.cluster.local
+          port: 11800
+        name: skywalking
+    defaultProviders:
+        tracing:
+        - "skywalking"
```

Comment:
I mean this part

Reply:
we can add L33-L37 to `demo` profile, just like 
https://github.com/istio/istio/blob/c42e0c02b9a192ee96b04777fd02db3632724a57/manifests/profiles/demo.yaml#L6

Reply:
> we can add L33-L37 to `demo` profile, just like
> https://github.com/istio/istio/blob/c42e0c02b9a192ee96b04777fd02db3632724a57/manifests/profiles/demo.yaml#L6

Thank you @zirain , I added this https://github.com/istio/istio/pull/40329

Reply:
Are we still waiting on https://github.com/istio/istio/pull/40329 to simplify this snippet?

Reply:
> Are we still waiting on [istio/istio#40329](https://github.com/istio/istio/pull/40329) to simplify this snippet?

This PR itself is complete and can be merged, if istio/istio#40329 can be merged I can simplify this PR. See my comment here https://github.com/istio/istio.io/pull/11683#issuecomment-1250315385

> Hi, I can't see any chance for now that https://github.com/istio/istio/pull/40329 can get merged so I end up using the `none` profile with an `IstioOperator` CR to enable the skywalking tracer in this PR, if that PR can be merged I might simplify this doc by just using the demo profile